app/models/teams/account.rb

```diff
@@ -18,22 +18,12 @@ class Teams::Account < ActiveRecord::Base
   has_many :plans, through: :account_plans
   belongs_to :admin, class_name: 'User'
 
-  validates :team_id, presence: true, uniqueness: true
   validates_presence_of :stripe_card_token
   validates_presence_of :stripe_customer_token
+  validates :team_id, presence: true, uniqueness: true
 
   attr_protected :stripe_customer_token, :admin_id
 
-  validate :stripe_customer_token, presence: true
```

Comment:
Isn't that unnecessary? As far as I understand - Teams who haven't paid, don't have Accounts/Plans. We only create Accounts/Plans for teams who **are** paying customers, so a vanilla `presence` validation makes sense. 

The only way I see a team going to an *inactive* state, is if they decided to cancel their subscription, in which case they should be moved to the free plan. In that case, they'll still have the old `stripe_customer_token`, so our validation still holds.